### PR TITLE
add inductor_train.sh

### PR DESCRIPTION
**Pull request by ESI-SYD**
add inductor_train.sh

### CODE REVIEW

File: scripts/modelbench/inductor_train.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        
+    elif [[ $CHANNELS == "last" ]]; then
+        # channels last
+        echo "Skip Channels last testing...."
+    else
+        echo "Please check channels foramt with first / last."
+    fi
+}
+
+# single-thread
```
Comment (chuanqi129) [inline scripts/modelbench/inductor_train.sh:61]: Please remove single thread test part from training test scripts.

File: scripts/modelbench/inductor_train.sh [hunk 2]
```diff
@@ -0,0 +1,112 @@
+export LD_PRELOAD=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}/lib/libiomp5.so:${CONDA_PREFIX:-"$(dirname $(which conda))/../"}/lib/libjemalloc.so
+export MALLOC_CONF="oversize_threshold:1,background_thread:true,metadata_thp:auto,dirty_decay_ms:-1,muzzy_decay_ms:-1"
+
+# multiple / single / all
+THREAD=${1:-multiple}
```
Comment (chuanqi129) [inline scripts/modelbench/inductor_train.sh:5]: Training test just has multiple thread mode

File: scripts/modelbench/inductor_train.sh (hunk 1 above)
Comment (ESI-SYD) [reply scripts/modelbench/inductor_train.sh]: Done

File: scripts/modelbench/inductor_train.sh (hunk 2 above)
Comment (ESI-SYD) [reply scripts/modelbench/inductor_train.sh]: Done

File: scripts/modelbench/inductor_train.sh [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    do
+        # Commands for timm_models for device=cpu, dtype=float32 for training and for performance testing
+        python -m torch.backends.xeon.run_cpu --node_id 0 benchmarks/dynamo/timm_models.py --performance --float32 -dcpu --training --inductor   --no-skip --dashboard --only ${timm_model} --cold_start_latency --output=${LOG_DIR}/multi_threads_cf_${timestamp}_perf.csv 2>&1 | tee -a ${LOG_DIR}/multi_threads_model_bench_log_${timestamp}.log
+        # Commands for timm_models for device=cpu, dtype=float32 for training and for accuracy testing
+        python -m torch.backends.xeon.run_cpu --node_id 0 benchmarks/dynamo/timm_models.py --accuracy --float32 -dcpu --training --inductor   --no-skip --dashboard --only ${timm_model} --output=${LOG_DIR}/multi_threads_cf_${timestamp}_acc.csv 2>&1 | tee -a ${LOG_DIR}/multi_threads_model_bench_log_${timestamp}.log
+    done       
+    
+elif [[ $CHANNELS == "last" ]]; then
+    # channels last
+    echo "Skip Channels last testing...."
```
Comment (chuanqi129) [inline scripts/modelbench/inductor_train.sh:50]: Why we need skip channels last test? For channels format can refer https://github.com/chuanqi129/inductor-tools/blob/main/scripts/modelbench/inductor_single_run.sh#L15-L18 to reduce redundant code

Comment (ESI-SYD) [reply scripts/modelbench/inductor_train.sh]: Thanks, added.

**Review by chuanqi129 — CHANGES_REQUESTED** [review]
LGTM, just have 1 minor change request to fix channels format hardcode in log file name

**Comment by ESI-SYD** [comment]
Done

**Review by chuanqi129 — APPROVED** [review]

**Pull request merged**